commander/test/bootstrapping/commands/keys/create.spec.ts

```diff
@@ -0,0 +1,291 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import * as cryptography from '@liskhq/lisk-cryptography';
+import * as Config from '@oclif/config';
+import * as fs from 'fs-extra';
+import * as readerUtils from '../../../../src/utils/reader';
+import { CreateCommand } from '../../../../src/bootstrapping/commands/keys/create';
+import { getConfig } from '../../../helpers/config';
+
+jest.mock('@liskhq/lisk-cryptography', () => ({
```

Comment:
Just a check, the coding guidelines on testing indicate that we should avoid using `jest.mock` as much as possible. Doesn't explain why but a heads up 🙂.

![image](https://user-images.githubusercontent.com/10962699/184833910-8a16a651-d2c6-48b5-864c-9f9621c505c1.png)

Reply:
Thanks for the heads up! I guess here it's unavoidable since there is probably no better way of spying on `lisk-cryptography` functions.